data/timesketch.conf

```diff
@@ -376,13 +376,19 @@ LLM_PROVIDER_CONFIGS = {
     #   $ pip3 install google-generativeai
     'nl2q': {
         'vertexai': {
-            'model': 'gemini-2.0-flash-001',
+            'model': 'gemini-2.0-flash',
             'project_id': '',
         },
     },
     'llm_summarize': {
         'aistudio': {
-            'model': 'gemini-2.0-flash-001',
+            'model': 'gemini-2.0-flash',
```

Comment:
2.5?

Reply:
Not for the summary. It turns out 2.5-flash is a bit too slow for the llm_summary feature, and we have not yet the option to set thinking budgets etc in our framework.